examples/ChilloutMix-2.json

```diff
@@ -0,0 +1,8 @@
+{
+    "prompt": "best quality, ultra high res, (photorealistic:1.4), man wearing black suit, close-up portrait, streets background",
+    "negative": "paintings, sketches, (worst quality:2), (low quality:2), (normal quality:2), lowres, normal quality, ((monochrome)), ((grayscale)), skin spots, acnes, skin blemishes, bad anatomy, (long hair:1.4), DeepNegative, (fat:1.2), lowres, bad anatomy, bad hands, text, error, missing fingers, extra digit, fewer digits, cropped, worstquality, low quality, normal quality, jpegartifacts, signature, watermark, username, blurry, bad feet, cropped, poorly drawn hands, poorly drawn face, mutation, deformed, worst quality, low quality, normal quality, jpeg artifacts, signature, watermark, extra fingers, fewer digits, extra limbs, extra arms, extra legs, malformed limbs, fused fingers, too many fingers, long neck, cross-eyed, mutated hands, polar lowres, bad body, bad proportions, gross proportions, text, error, missing fingers, missing arms, missing legs, extra digit",
```

Comment:
follow the schema https://github.com/heurist-network/heurist-models/blob/main/examples/AAMXLAnimeMix-3.json
neg_prompt
width
height
num_inference_steps
guidance_scale instead of CFG
copy the sampler field as well